docs/faq/help.txt

```diff
@@ -21,7 +21,6 @@ Then, please post it in one of the following channels:
 
 * The Django Forum section `"Using Django"`_. This is for web-based
   discussions.
-* The |django-users| mailing list. This is for email-based discussions.
```

Comment:
Should we remove the reference django-users in README.rst (or at least the bit about "joining")?

Reply:
Yes, thank you! Updated
I missed this for the IRC ticket also (https://github.com/django/django/pull/19189)